tests/test-json-responses.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Test case for Basic Router responses.
+ *
+ * @group ajax
+ */
+class WPGraphQL_Test_Router extends WP_Ajax_UnitTestCase {
```

Comment:
@AramZS looks like the issue is that you used the same class name here as we have here: https://github.com/wp-graphql/wp-graphql/blob/develop/tests/test-router.php#L3

Try changing this class name to something like `WP_GraphQL_Test_JSON_Responses`